features/consent/modal.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Modal Fallback
+  The modal fallback pop-up displays when some checkbox agreement elements
+  are set to 'no' (i.e. unchecked).
+
+  Scenario: User sees the modal fallback when some checkbox agreement elements aren't checked
+    Given I do not exist as a user
+    And A study code exists
+    When I click on Register
+    And I fill in the user details
+    Then I should see the step one of consent
+    When I click on Next
+    Then I should see the step two of the consent
+    When I click all the checkboxes
```

Comment:
Just making sure I understand. Here you're first selecting all check boxes and then, with one more click, de-selecting the last one to trigger the fallback popup, right? Makes sense 👍

Reply:
Yep